actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -151,31 +142,20 @@ private void courseProgressMetricsV2(Request actorMessage) {
     List<Map<String, Object>> userData = new ArrayList<>();
     for (Map<String, Object> esContent : esContents) {
       Map<String, Object> map = new HashMap<>();
-      map.put(JsonKey.USER_NAME, (String) esContent.get(JsonKey.FIRST_NAME));
-      String phone = null;
-      if (esContent.containsKey(JsonKey.ENC_PHONE)) {
-        phone = decryptAndMaskPhone((String) esContent.get(JsonKey.ENC_PHONE));
-      }
-      map.put(JsonKey.PHONE, phone);
+      map.put(JsonKey.USER_NAME, esContent.get(JsonKey.NAME));
+      map.put(JsonKey.MASKED_PHONE, esContent.get(JsonKey.MASKED_PHONE));
       map.put(JsonKey.ORG_NAME, esContent.get(JsonKey.ROOT_ORG_NAME));
-      for (Map<String, Object> batchMap :
-          (List<Map<String, Object>>) esContent.get(JsonKey.BATCHES)) {
-        if (batchId.equalsIgnoreCase((String) batchMap.get(JsonKey.BATCH_ID))) {
-          calculateCourseProgressPercentage(batchMap, leafNodeCount);
-          formatEnrolledOn(batchMap);
-          map.putAll(batchMap);
-        }
-      }
+      map.put(JsonKey.PROGRESS, esContent.get(JsonKey.COMPLETED_PERCENT));
+      map.put(JsonKey.ENROLLED_ON, esContent.get(JsonKey.ENROLLED_ON));
       userData.add(map);
     }
 
-    courseProgressResult.put(JsonKey.COUNT, result.get(JsonKey.COUNT));
+    courseProgressResult.put(JsonKey.COUNT, courseBatchResult.get(JsonKey.PARTICIPANT_COUNT));
     courseProgressResult.put(JsonKey.DATA, userData);
     courseProgressResult.put(JsonKey.START_DATE, courseBatchResult.get(JsonKey.START_DATE));
     courseProgressResult.put(JsonKey.END_DATE, courseBatchResult.get(JsonKey.END_DATE));
     courseProgressResult.put(
-        JsonKey.COMPLETED_COUNT,
-        getCompletedCount(batchId, leafNodeCount, (Long) result.get(JsonKey.COUNT)));
+        JsonKey.COMPLETED_COUNT, courseBatchResult.get(JsonKey.COMPLETED_COUNT));
     Response response = new Response();
     response.put("response", "SUCCESS");
```

Comment:
Don't hard code "response" and "SUCCESS" String use some predefined constant or JsonKey